src/relay/relay.py

```diff
@@ -304,7 +305,48 @@ def get_earned_mediation_fees(
     def get_debt_list_of_user(self, user_address):
 
         event_selector = self.get_ethindex_db_for_currency_network()
-        return EventsInformationFetcher(event_selector).get_debts_lists(user_address)
+        debts_list_in_all_currency_networks = EventsInformationFetcher(
+            event_selector
+        ).get_debts_lists(user_address)
+
+        # Add the max claimable value and path to debts and return them as a dict
+        enriched_debts_lists_in_all_currency_networks = []
+        for currency_network in debts_list_in_all_currency_networks.keys():
+            enriched_debts_list = []
+
+            for debtor in debts_list_in_all_currency_networks[currency_network].keys():
+                debt = debts_list_in_all_currency_networks[currency_network][debtor]
+                enriched_debt = {
+                    "debtor": debtor,
+                    "value": debt,
+                }
+
+                if debt > 0:
+                    debt_claimer = user_address
+                    debt_payer = debtor
+                elif debt < 0:
+                    debt_claimer = debtor
+                    debt_payer = user_address
+                else:
+                    raise RuntimeError(f"Found null debt with debtor {debtor}")
+                capacity_path = self.currency_network_graphs[
+                    currency_network
+                ].find_maximum_capacity_path(
+                    debt_payer, debt_claimer, timestamp=int(time.time())
+                )
+                enriched_debt["maximum_claimable_value"] = min(
+                    capacity_path.capacity, abs(debt)
+                )
+                if capacity_path.path != []:
+                    enriched_debt["claim_path"] = capacity_path.path
+
+                enriched_debts_list.append(enriched_debt)
+
+            enriched_debts_lists_in_all_currency_networks.append(
+                {"currency_network": currency_network, "debts": enriched_debts_list}
+            )
```

Comment:
Move this block to a separate function. addAdditionalDebtInformation or I don't know what.

Then your code would read like poetry :)
...
get_debt_list_of_user
first fetch the user's debts list
for each debt add debtor, value and path

the end...

And the person reading on the other hand wouldn't have to go into the internal details of the for loop, but will still know what you do there :)

Reply:
I didn't want to do that because the structure of `debts_list_in_all_currency_networks` is kind of implicit, it depends on `EventsInformationFetcher.get_debts_lists`. And the structure of the output of this function depends on the schema `DebtsListInCurrencyNetwork` (or the other way around if you will). So I didn't want to split it and separate even more the responsibilities while it is actually a bit entangled.

Reply:
I'll rework it.